### ISSUE TITLE
Solvers

### ISSUE THREAD

**Issue by amirsani** [pan5431333/pyoptree#4]
Is it possible to use open source solvers? If so, can you post an example on how?

Thank you

**Comment by pan5431333** [comment]
Yes I use Pyomo as the modeling language, which means you can use any solver that Pyomo supports. I'll try to give an example this Saturday or Sunday. Thanks for asking.

**Comment by amirsani** [comment]
Thanks, that would be helpful
